draft-ietf-lamps-kyber-certificates.md

```diff
@@ -421,14 +493,21 @@ This appendix contains examples of ML-KEM public keys, private keys and
 certificates.
 
 
-## Example Private Key {#example-private}
+## Example Private Keys {#example-private}
 
 The following examples show ML-KEM private keys in different formats,
 all derived from the same seed `000102…1e1f`. For each security level,
 we show the seed-only format (using a context-specific `[0]` primitive
 tag with an implicit encoding of OCTET STRING), the expanded format,
 and both formats together.
 
+NOTE: All examples use the same seed value, showing how the same seed	
+produces different expanded keys for each security level.  The seed-	
+only format is the most compact representation and is RECOMMENDED for	
+storage.  The expanded format might be used in constrained	
+environments where key expansion is not desired, and the both format	
+allows for key validation during import.
```

Comment:
```suggestion
produces different expanded keys for each security level.
```

This change would align with the ML-DSA [PR](https://github.com/lamps-wg/dilithium-certificates/pull/98).